### PR TITLE
fix: fix for opcua issues eclipse/thingweb.node-wot#410. Node-opcua l…

### PR DESCRIPTION
**Pull request by lukesmolo**
Fix for OPCUA issues #410

### CODE REVIEW

File: examples/servients/opcua-cli/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@node-wot/opcua-cli",
-  "version": "0.7.2",
+  "version": "0.7.6",
```
Comment (danielpeintner) [inline examples/servients/opcua-cli/package.json:3]: We are about to move to "0.7.7" ... can you update this as well. Thanks!

File: examples/servients/opcua-cli/package.json [hunk 2]
```diff
@@ -18,16 +18,16 @@
...
     "wot-typescript-definitions": "0.7.3"
   },
   "dependencies": {
-    "@node-wot/core": "0.7.2",
-    "@node-wot/binding-http": "0.7.2",
-    "@node-wot/binding-file": "0.7.2",
-    "@node-wot/binding-opcua": "0.7.2",
+    "@node-wot/binding-file": "0.7.5",
+    "@node-wot/binding-http": "0.7.5",
+    "@node-wot/binding-opcua": "0.7.5",
```
Comment (danielpeintner) [inline examples/servients/opcua-cli/package.json:29]: here it should be "0.7.6"

**Comment by danielpeintner** [comment]
Thanks @lukesmolo 
I noted that an OPCUA test was failing..

```
  1) OPCUA client test
       should read a property:

      AssertionError: expected 2 to equal 1
      + expected - actual

      -2
      +1
```

File: examples/servients/opcua-cli/package.json (hunk 2 above)
Comment (lukesmolo) [reply examples/servients/opcua-cli/package.json:29]: ok

File: examples/servients/opcua-cli/package.json (hunk 1 above)
Comment (lukesmolo) [reply examples/servients/opcua-cli/package.json:3]: ok, sure

**Comment by lukesmolo** [comment]
> Thanks @lukesmolo
> I noted that an OPCUA test was failing..
> 
> ```
>   1) OPCUA client test
>        should read a property:
> 
>       AssertionError: expected 2 to equal 1
>       + expected - actual
> 
>       -2
>       +1
> ```
Damn, I got it, I think there is a race condition since for instance, I cannot see the same issue in my local environment or also [here](https://github.com/lukesmolo/thingweb.node-wot/runs/2951271996). The _write Property_ arrives before the _read_  one. Anyway, let me fix it.

File: examples/servients/opcua-cli/package.json (hunk 2 above)
Comment (relu91) [reply examples/servients/opcua-cli/package.json:29]: > here it should be "0.7.6"

Did you mean 0.7.7 ?

Comment (danielpeintner) [reply examples/servients/opcua-cli/package.json:29]: In  this section we seem to be sticking to the _released_ version that currently is 0.7.6
Maybe it is even easier to have something like `0.7.x` here

**Review by relu91 — APPROVED** [review]

**Review by danielpeintner — APPROVED** [review]

**Comment by danielpeintner** [comment]
@sebastiankb since you raised https://github.com/eclipse/thingweb.node-wot/issues/410
Can you have a look

**Comment by sebastiankb** [comment]
@lukesmolo many thanks. I will test this PR with a S7 and let you know if its working now.

**Review by sebastiankb — APPROVED** [review]

**Comment by sebastiankb** [comment]
Just test the new version with the S7 and indeed, seems to work now :-)

@lukesmolo many thanks again

**Pull request merged**